src/game/player.gd

```diff
@@ -1,42 +1,47 @@
 extends CharacterBody2D
 
 
-@export var SPEED = 150.0
-@export var DASH_SPEED = 300.0
-@export var MAX_ACCELERATION = 400.0
-@export var MAX_AIR_ACCELERATION = 350.0
-@export var MAX_DECELERATION = 600.0
-@export var MAX_AIR_DECELERATION = 350.0
-@export var MAX_TURN_SPEED = 1500.0
-@export var MAX_AIR_TURN_SPEED = 1200.0
-@export var MAX_DASH_ACCELERATION = 4000.0
-@export var MAX_DASH_DECELERATION = 3800.0
-@export var MAX_DASH_TIME = 0.2
-@export var DASH_COOLDOWN = 0.3
-@export var JUMP_VELOCITY = -300.0
-@export var COYOTE_TIME_LIMIT = 0.2
-@export var LANDING_JUMP_BUFFER_TIME = 0.05
+@export var SPEED := 150.0
+@export var DASH_SPEED := 300.0
+@export var MAX_ACCELERATION := 400.0
+@export var MAX_AIR_ACCELERATION := 350.0
+@export var MAX_DECELERATION := 600.0
+@export var MAX_AIR_DECELERATION := 350.0
+@export var MAX_TURN_SPEED := 1500.0
+@export var MAX_AIR_TURN_SPEED := 1200.0
+@export var MAX_DASH_ACCELERATION := 4000.0
+@export var MAX_DASH_DECELERATION := 3800.0
+@export var MAX_DASH_TIME := 0.2
+@export var DASH_COOLDOWN := 0.3
+@export var GROUNDED_JUMP_VELOCITY := -300.0
+@export var AERIAL_JUMP_VELOCITY := -280.0
+@export var JUMP_RELEASE_VELOCITY_REDUCTION := 3.0
+@export var COYOTE_TIME_LIMIT := 0.2
+@export var LANDING_JUMP_BUFFER_TIME := 0.05
```

Comment:
i expect a full written justification for why you changed these values smh.

Reply:
me -> 😨  🔫  😈  <- terry

Reply:
yes